### PR TITLE
Adapt methods in HBaseRequestAdapter

### PR DESCRIPTION
**Pull request by elisheva-qlogic**
`adapt(Put put)` and `adapt(Delete delete)` changed to return RowMutation instead of MutateRowRequest

### CODE REVIEW

**Comment by sduskis** [comment]
@elisheva-qlogic, can you please resolve the conflicts?

**Comment by sduskis** [comment]
In the adapter, please also convert `adapt(RowMutations)` to return `Mutations`.

File: bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/PutAdapterPerf.java [hunk 1]
```diff
@@ -44,6 +47,7 @@ public static void main(String[] args) {
     HBaseRequestAdapter adapter =
         new HBaseRequestAdapter(options,
             TableName.valueOf("tableName"), new Configuration());
+    requestContext = RequestContext.create(adapter.getBigtableTableName().toGcbInstanceName(), "");
```
Comment (sduskis) [inline bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/PutAdapterPerf.java]: `adapter.getBigtableTableName().toGcbInstanceName()` doesn't seem right here.  Perhaps `options.getGcbInstanceName()`?

File: bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/PutAdapterPerf.java [hunk 2]
```diff
@@ -68,8 +72,8 @@ private static void putAdapterPerf(HBaseRequestAdapter adapter, Put put) {
     { 
       long start = System.nanoTime();
       for (int i = 0; i < count; i++) {
-        MutateRowRequest adapted = adapter.adapt(put);
-        BigtableGrpc.getMutateRowMethod().streamRequest(adapted);
+        RowMutation adapted = adapter.adapt(put);
+        BigtableGrpc.getMutateRowMethod().streamRequest(adapted.toProto(requestContext));
```
Comment (sduskis) [inline bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/PutAdapterPerf.java]: This seems wrong for the perf class.  I'll work with you on updating this class in a different PR.

File: bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/PutAdapterPerf.java [hunk 3]
```diff
@@ -80,9 +84,9 @@ private static void putAdapterPerf(HBaseRequestAdapter adapter, Put put) {
     { 
       long start = System.nanoTime();
       for (int i = 0; i < count; i++) {
-        MutateRowRequest adapted = adapter.adapt(put);
-        adapted.getSerializedSize();
-        BigtableGrpc.getMutateRowMethod().streamRequest(adapted);
+        RowMutation adapted = adapter.adapt(put);
+        adapted.toProto(requestContext).getSerializedSize();
```
Comment (sduskis) [inline bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/PutAdapterPerf.java]: Please only do the `toProto(requestContext)` once.

File: bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/PutAdapterPerf.java (hunk 2 above)
Comment (elisheva-qlogic) [reply bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/PutAdapterPerf.java]: As discussed, this change is ok

**Comment by sduskis** [comment]
Thanks for the PR!

**Pull request merged**